src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BindingBuildTest.cs

```diff
@@ -227,7 +227,10 @@ public void BindingCustomJavaApplicationClass ()
 				IsRelease = true,
 			};
 			binding.AndroidClassParser = "class-parse";
-			binding.Jars.Add (new AndroidItem.EmbeddedJar (() => "$(MonoDroidInstallDirectory)\\lib\\mandroid\\android-support-multidex.jar"));
+			var multidexJar = Environment.OSVersion.Platform == PlatformID.Win32NT
+				? Path.Combine (Environment.GetFolderPath (Environment.SpecialFolder.ProgramFilesX86), "MSBuild", "Xamarin", "Android", "android-support-multidex.jar") 
```

Comment:
The "proper" mechanism is to [use the `XAMARIN_ANDROID_REGKEY` environment variable](https://github.com/xamarin/xamarin-android/blob/70d9e2f/src/Xamarin.Android.Build.Tasks/MSBuild/Xamarin/Xamarin.Android.Sdk.props#L50). This is how Visual Studio communicates to our build system where things are located...ish.

[Related](https://github.com/xamarin/xamarin-android/blob/0af965e/src/Xamarin.Android.Build.Utilities/Sdks/AndroidSdkWindows.cs#L29)?